xds/src/main/java/io/grpc/xds/WrrLocalityLoadBalancer.java

```diff
@@ -88,13 +107,13 @@ public boolean acceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
               wrrLocalityConfig.childPolicy));
     }
 
-    // Remove the locality weights attribute now that we have consumed it. This is done simply for
+    // Remove the locality weight attribute now that we have consumed it. This is done simply for
     // ease of debugging for the unsupported (and unlikely) scenario where WrrLocalityConfig has
     // another wrr_locality as the child policy. The missing locality weight attribute would make
```

Comment:
these all seem not relevant anymore? because the new attribute is never set on resolvedaddress but on individual eag.
Although you might want to remove `ATTR_LOCALITY_WEIGHT ` instead (for what the comment is)?

Reply:
I think I did mean to remove the attribute from each eag, but then simply forgot. I was going to add that logic now, but I realized the list of eags in ResolvedAddresses is immutable as is EquivalentAddressGroup itself, meaning I can't remove it (maybe I'm missing something?).

Since this was a debugging optimization for a very unlikely scenario (wrr_locality under a wrr_locality) I opted to just removing this bit.